include/logging/log.h

```diff
@@ -136,45 +136,49 @@ extern "C" {
  * @details It's meant to report severe errors, such as those from which it's
  * not possible to recover.
  *
- * @param data   Pointer to the data to be logged.
- * @param length Length of data (in bytes).
+ * @param _str    Raw string (without format specifiers).
```

Comment:
1. Why I cannot specify string with format codes?
2. IMHO it might be worth mentioning that the string have to be placed in persistent memory.

Reply:
string with format specifiers would require standard message (at least additional 32 bytes in the message) and simply put - more work. Since this feature is not required now i would like wait with that until there is a requirement.  On the other hand, one thing we could do is to change that API from LOG_HEXDUMP(str, data, len) to LOG_HEXDUMP(data, len, str). That opens the door in the future to be extended to LOG_HEXDUMP(data, len, fmt,...). I didn't do that initially because logically string appears before data (e.g. LOG_HEXDUMP_INF("received packet:", data, len)). i can flip that.

Reply:
I don't require formatting support. Basing on the description I understood that for some reason I cannot specify strings like "%s". If this is true, it is serious weakness.

If I can specify "%s" and it will be not interpreted but printed as "%s", then the only change I will request is updating comment (removing: `(without format specifiers)`).

Reply:
done, string is not formatted, but printed as is. (removed: (without format specifiers))